packages/quantic/force-app/examples/main/lwc/exampleQuanticRefineModalContent/exampleQuanticRefineModalContent.js

```diff
@@ -15,6 +16,12 @@ export default class ExampleQuanticRefineModalContent extends LightningElement {
       description: 'Whether the Quantic Sort component should be hidden.',
       defaultValue: false,
     },
+    {
+      attribute: 'disableFacetOrdering',
```

Comment:
You are not passing this property in the HTML file of the ExampleQuanticRefineToggle component.
I wonder, how the E2E of this component are passing?